unreal_msgs/StaticMeshDescription[] static_mesh_descriptions
---
bool success
